sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/model/AccessToken.java

```diff
@@ -57,18 +59,62 @@ public String getAccessToken() {
     /**
      * Set the access token.
      *
-     * @param accessToken the access token.
+     * @param accessToken The access token.
      */
     public void setAccessToken(String accessToken) {
         this.accessToken = accessToken;
     }
 
     /**
-     * Reserve 60 seconds, in case that the time the token is used it is valid but when the token gets to the server side, it expires.
-     * @return boolean, whether the token is expired.
+     * Indicates whether the token is expired or not.
+     *
+     * <p>Reserve 60 seconds, in case that the time the token is used when it is still valid but when it gets to the
+     * server side, it expires.</p>
      *
+     * @return A value indicating whether the token is expired or not.
      */
     public boolean isExpired() {
         return OffsetDateTime.now().isAfter(creationDate.plusSeconds(expiresIn - 60));
     }
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
+        jsonWriter.writeStringField("accessToken", this.accessToken);
+        jsonWriter.writeNumberField("expiresIn", this.expiresIn);
```

Comment:
This changed the names of the JSON properties from `access_token` and `expires_in` to `accessToken` and `expiresIn`, this should be changed back to the old names. Same issue is present in the deserialization function.